@here/harp-mapview/lib/Utils.ts

```diff
@@ -239,9 +239,143 @@ export namespace MapViewUtils {
             groundPlane.constant = 0;
             groundSphere.radius = EarthConstants.EQUATORIAL_RADIUS;
         }
+        if (result !== null) {
+            return result;
+        }
+        return getGazingTargetFromCamera(camera, projection, elevation);
+    }
+
+    function getGazingTargetFromCamera(
```

Comment:
I still don't see why we need to spend so much effort for the target point. I  agree that we should do sth. like that for the `MapControls` but why for the target point?

Reply:
The assumption here (and I guess easiest one) it that focus point based rotation should be based on the same target point as calculated in MapView. Of course we could introduce second 
_focus_ point just for MapControls, but I think it makes stuff over-complicated what do you think?